### DISCUSSION TITLE
Redirect on error

### DISCUSSION THREAD

**Discussion by Wulfheart** [livewire/livewire#1217] [4. General]
![grafik](https://user-images.githubusercontent.com/25671390/86519207-efde7100-be38-11ea-8a3d-72b7328ac06d.png)

Is it possible to redirect on an error to avoid this error popup? I'd like to redirect to an error page which displays the corresponding error code.

**Comment by nuernbergerA** [comment]
https://github.com/livewire/livewire/pull/1146

```js
window.livewire.onError(statusCode => {
    //
})
```

**Reply by Wulfheart** [reply]
Thanks!
